exercises/01-rest-api.md

```diff
@@ -2,19 +2,19 @@
 
 ## Learning Goals
 
-- You know how to make basics of making REST API requests with Robot Framework.
+- You grasp the basics of making REST API requests with Robot Framework.
 
 ## Introduction
 
 REST API based testing or RPA can be done in multiple different ways in Robot Framework.
-You can of course do this with your own custom library using `requests`
+You can of course do this with your own custom library using Python's `requests`
```

Comment:
Maybe add link to the documentation as well:
```suggestion
You can of course do this with your own custom Robot Framework library using 3rd-party [`requests`](https://docs.python-requests.org/en/latest/) library
```

Reply:
Good idea.